plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/OpenShiftConnector.java

```diff
@@ -1318,9 +1319,15 @@ private ContainerInfo createContainerInfo(Service svc,
         info.setNetworkSettings(networkSettings);
         info.setHostConfig(hostConfig);
         info.setImage(imageInfo.getConfig().getImage());
+        info.setState(getContainerState(pod));
         return info;
     }
 
+    private ContainerState getContainerState(final Pod pod) {
+        ContainerState state = new ContainerState();
+        state.setStatus(pod.getStatus().getPhase());
```

Comment:
there is also a possibility to get ContainerState[1] which can be:
```
     * @param running
     * @param waiting
     * @param terminated
```
this model is not also aligned with docker one

[1] http://static.javadoc.io/io.fabric8/kubernetes-api/2.0.38/io/fabric8/kubernetes/api/model/ContainerState.html